README.md

```diff
@@ -112,12 +112,18 @@ for your dataset.
 `deploy.json` will still need to be filled out. A temporary file will be written to a
 GCS bucket in the project in `deploy.json`, even for local deployment of Data
 Explorer. Choose a project where you have at least Project Editor permissions.
-* `deploy/create-export-url-bucket.sh DATASET`, where `DATASET` is the name of
+* Create export bucket. This only needs to be done once per deploy project.  
+`deploy/create-export-url-bucket.sh DATASET`, where `DATASET` is the name of
```

Comment:
Nit maybe: "Run `deploy/create-export-url-bucket.sh DATASET` from the root of the repo, where ..."

Reply:
Done